bundles/org.openhab.binding.salus/src/test/java/org/openhab/binding/salus/internal/cloud/rest/HttpSalusApiTest.java

```diff
@@ -216,15 +224,15 @@ public void testSetValueForPropertyWithInvalidAuthTokenThrowsHttpUnauthorizedExc
         when(restClient.post(anyString(), any(), any()))
```

Comment:
Missed this in previous PR, but the line above is not used. Please use or remove.

Reply:
done